cpp/ql/src/Security/CWE/CWE-611/XXE.ql

```diff
@@ -217,12 +224,75 @@ class SetFeatureTranformer extends XXEFlowStateTranformer {
 }
 
 /**
- * The `AbstractDOMParser.parse`, `SAXParser.parse` or `SAX2XMLReader.parse`
- * method.
+ * The `DOMLSParser.getDomConfig` function.
+ */
+class GetDomConfig extends Function {
+  GetDomConfig() {
+    this.hasName("getDomConfig") and
+    this.getDeclaringType() instanceof DomLSParserClass
```

Comment:
We can use `getClassAndName` as well here, right?

Reply:
Yep, you have a good eye for spotting these.  Updated.